nncf/torch/composite_compression.py

```diff
@@ -107,6 +109,16 @@ def initialize(self, model: ModelType) -> None:
     def _get_transformation_layout(self, target_model: NNCFNetwork) -> PTTransformationLayout:
         pass  # Higher-level get_transformation_layout is overridden, no need to define this
 
+    def load_state(self, state: Dict[str, Any], version: PTCompressionStateVersion = None) -> None:
```

Comment:
Because of inheritance order `PTCompositeCompressionAlgorithmBuilder` prioritize `CompositeCompressionAlgorithmBuilder` methods which declare `load_state` method without version, so explicit override is needed.
Actually we could change order of inheritance order since none of other `PTCompositeCompressionAlgorithmBuilder` methods have conflicting implementations (so `load_state` won't have to be overloaded) but this would lead to explicit calling of `CompositeCompressionAlgorithmBuilder.__init__` instead of `super().__init__`
Same for `PTCompositeCompressionAlgorithmController`

Reply:
What about introducing version in the common part? It would also align compression state with TF and reduce the need for tricky overriding and order of inheritance